src/java.desktop/share/native/libfontmanager/freetypeScaler.c

```diff
@@ -187,6 +195,8 @@ static unsigned long ReadTTFontFileFunc(FT_Stream stream,
                                           scalerInfo->font2D,
                                           sunFontIDs.ttReadBlockMID,
                                           bBuffer, offset, numBytes);
+            // This is a callback, we are not returning immediately to Java and better report exceptions now
+            CHECK_EXCEPTION(env);
```

Comment:
Probably we should report it only if "debugFonts" was set?

Reply:
Done.